www/docs/customization/announce.md

```diff
@@ -165,3 +171,43 @@ announce:
     # Defaults to `{{ .ProjectName }} {{ .Tag }} is out!`
     title_template: ''GoReleaser {{ .Tag }} was just released!''
 ```
+
+## SMTP
+
+For it to work, you'll need to set some environment variables on your pipeline:
+
+- `SMTP_PASSWORD`
+
+Then, you can add something like the following to your `.goreleaser.yml` config:
+
+```yaml
+# .goreleaser.yml
+announce:
+  smtp:
+    # Wether its enabled or not.
+    # Defaults to false.
+    enabled: true
+
+    # SMTP Host
+    host: "smtp.gmail.com"
+
+    # SMTP Port
+    port: 587
+
+    # Sender of the email
+    from: ""
+
+    # Receiver of the email
+    to: ""
```

Comment:
this should be in array type:

```yaml
to:
 - foo
 - bar
```

Reply:
thank you @Dentrax, it should be fixed now.